style.css

```diff
@@ -0,0 +1,30 @@
+html {
+    background-color: black;
+}
+
+h1 {
+    font-family:Arial, Helvetica, sans-serif;
+    color: white;
+    width: 580px;
+    height: 48px;
+    top: 252px;
+    left: 236px;
+}
+
+h5 {
+    font-family:Arial, Helvetica, sans-serif;
+    color: grey;
+    width: 294px;
+    height: 22px;
+    top: 300px;
+    left: 236px;
+}
+
+p {
+    font-family:Arial, Helvetica, sans-serif;
```

Comment:
font-family gets repetitive throughout. we can probably wrap this in the `<body>`. I do recommend, how the styles of the text was fetched in #15 or #16